web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/var/document.js

```diff
@@ -0,0 +1,5 @@
+define( function() {
```

Comment:
:no_entry_sign: **Error**: Opening parenthesis of a multi-line function call must be the last content on the line (*PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket*).